pkg/operator/ceph/nvmeof_recoverer/nvmeofstorage/controller.go

```diff
@@ -190,12 +191,28 @@ func (r *ReconcileNvmeOfStorage) reconcile(context context.Context, request reco
 					return reconcile.Result{}, nil
 				}
 				logger.Debugf("next host found: %s", nextHostName)
-				// TODO: Add create and run job for nvme-of device switch to the next host
-				// Placeholder for the job creation and execution
+
```

Comment:
Before allocating nvme-oF storage to the new host, I think we need to check if the old location is removed correctly, if not, we should umount the original path if you have time to reconsider this.

Reply:
That's a good feedback. Since we need to disconnect after the `OSD Delete` is complete.
I will add a code this after moving the `transfer-related code` from `osd.go`.